### PR TITLE
Add backspace key handling

### PR DESCRIPTION
**Pull request by nxnfufunezn**

### CODE REVIEW

File: src/src/lib.rs [hunk 1]
```diff
@@ -26,6 +26,5 @@ pub extern "C" fn kmain() -> ! {
 
     vga::clear_console();
     kprintln!("Kernel initialized.");
-
```
Comment (steveklabnik) [inline src/src/lib.rs]: can we keep this newline? I like this newline.

**Comment by steveklabnik** [comment]
Awesome. One small formatting nit. Also, thanks for converting to spaces, I missed that originally

File: src/src/lib.rs (hunk 1 above)
Comment (nxnfufunezn) [inline src/src/lib.rs]: :laughing: ok

**Comment by nxnfufunezn** [comment]
@steveklabnik I am pushing this dummy error code for proper alignment but it page faults?
I don't know what I am doing wrong..
Can you take a look, please?
https://github.com/nxnfufunezn/RstyOS/blob/fix-idt/src/interrupts/idt.rs#L54-L62

**Comment by nxnfufunezn** [comment]
rebased!

**Comment by steveklabnik** [comment]
Unfortunately, this code is gone for now, so this can't get merged. Thank you! And when we get it back, maybe we can apply it then :smile:

**Pull request closed without merging**